### PR TITLE
Deactivate macOS universal build

### PR DESCRIPTION
**Pull request by capital-G**
## Purpose and Motivation

A quick fix to get our pipeline working again, see https://github.com/supercollider/supercollider/issues/6352. This must be a temporary solution as we currently do not have a native build for Apple ARM computers, but it helps us to get our build and testing pipeline up and running again.

* Disable universal build
* Switching to x64-legacy binary for testing on macOS

## Types of changes

- Bug fix

## To-do list

- [x] Code is tested
- [ ] All tests are passing
- [ ] Updated documentation
- [ ] This PR is ready for review

### CODE REVIEW

**Review by JordanHendersonMusic — APPROVED** [review]
This all looks good, and thanks for the quick patch!

Could you make a high priority issue (can they be pinned to the top?) to remove this temporary fix after it's been merged - don't want to forget about it!

**Comment by capital-G** [comment]
> This all looks good, and thanks for the quick patch!
> 
> Could you make a high priority issue (can they be pinned to the top?) to remove this temporary fix after it's been merged - don't want to forget about it!

Please do not merge yet, I am running another test which has not such a big impact but the pipeline is still running.

**Pull request closed without merging**
